src/components/mods/modsTable.tsx

```diff
@@ -330,148 +436,159 @@ export const ModsTable = ({ qualities, difficulties, modsWithInfo, isLoading }:
 
     const { cx, classes } = useStyles();
 
+    const tabColors = [classes.tab1, classes.tab2, classes.tab3, classes.tab4, classes.tab5];
+
     return (
-        <DataTable
-            classNames={{ root: classes.table, pagination: classes.pagination }}
-            defaultColumnProps={{
-                cellsClassName: (record) => {
-                    return cx(
-                        classes.modCell,
-                        record.isExpanded && classes.expandedModCell,
-                    );
-                },
-            }}
-            height={550}
-            striped
-            textSelectionDisabled
-            withColumnBorders
-            highlightOnHover
-            fetching={isLoading}
-            records={records}
-            idAccessor={(record) => record.id}
-            columns={[
+        <div>
+            <div className={classes.tabContainer}>
```

Comment:
It's better to use a fragment unless you actually *need* a div. This way the rendered html won't contain an unnecessary element. This is marginally more efficient, but more importantly it's more semantically correct which avoids some problems encountered by screen readers (accessibility stuff).